### PR TITLE
add container env vars for GPUs

### PR DESCRIPTION
**Pull request by malek-delphai**

### CODE REVIEW

File: charts/delphai-keda/templates/deployment.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ spec:
   selector:
     matchLabels:
       app: {{ .Release.Name }}
-    
+
```
Comment (anton-delphai) [inline charts/delphai-keda/templates/deployment.yml:14]: :+1: :smile:

File: charts/delphai-keda/templates/deployment.yml [hunk 2]
```diff
@@ -68,6 +68,13 @@ spec:
             - name: {{ .name }}
               value: {{ .value }}
             {{- end }}
+            {{ if .Values.gpu }}
+            - name: NVIDIA_VISIBLE_DEVICES
+              value: "all"
+            - name: NVIDIA_DRIVER_CAPABILITIES
+              value: "compute,utility"
```
Comment (anton-delphai) [inline charts/delphai-keda/templates/deployment.yml:75]: Yes, it should be like that, check this https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/user-guide.html#driver-capabilities

**Review by anton-delphai — COMMENTED** [review]
One missing thing — you need to bump the version in `Chart.yaml`, otherwise the CI won't be able to publish new chart

File: charts/delphai-keda/templates/deployment.yml (hunk 1 above)
Comment (malek-delphai) [reply charts/delphai-keda/templates/deployment.yml:14]: This should probably be my text editor 😄

Comment (anton-delphai) [reply charts/delphai-keda/templates/deployment.yml:14]: Yes, and I like this cleanup

**Review by anton-delphai — APPROVED** [review]
Looks good!
Go ahead and merge if when you'll confirm it's working

**Pull request merged**